test/e2e-kind.sh

```diff
@@ -115,21 +121,29 @@ install_dummylb() {
     echo
 }
 
+install_reloader() {
+    echo 'Installing reloader...'
+    curl -sL https://raw.githubusercontent.com/stakater/Reloader/master/deployments/kubernetes/reloader.yaml |
```

Comment:
Should we get the latest release instead of pulling master?

Reply:
hm but the update the reloader.yaml with every release as far as I have seen

Reply:
:thinking: I'm still kinda thinking it would be good to land on a release rather than master.
Here's my suggestion:
```suggestion
    RELOADER_LATEST_RELEASE="$(curl -s https://api.github.com/repos/stakater/Reloader/releases/latest | jq -r '.tag_name')"
    curl -sL https://raw.githubusercontent.com/stakater/Reloader/${RELOADER_LATEST_RELEASE}/deployments/kubernetes/reloader.yaml |
```
Let me know what you think? :thinking:

Reply:
sure thing this will work
but we also need to make sure that `jq` is present in this case

Reply:
decided to go for awk here

Reply:
Yep, np. ty!